### PR TITLE
Repackaging

### PR DESCRIPTION
**Pull request by turian**
Remove all the hear-eval cruft.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,124 +1,35 @@
-# hear2021-eval-kit
+# hear-preprocess
 
-Evaluation kit for HEAR 2021 NeurIPS competition
+Dataset preprocessing code for the HEAR 2021 NeurIPS competition.
+
+Unless you are a HEAR organizer or want to contribute a task,
+you won't need this repo. Use
+[hear-eval-kit](https://github.com/neuralaudio/hear-eval-kit/) to
+evaluate your embedding models on these tasks.
```
Comment (jorshi) [inline README.md:8]: Are we going to host these datasets somewhere for participants to use?

File: hearpreprocess/pipeline.py [hunk 2]
```diff
@@ -10,13 +10,14 @@
...
-import heareval.tasks.util.audio as audio_util
 import luigi
 import pandas as pd
-from heareval.tasks.util.luigi import WorkTask, download_file, new_basedir
 from slugify import slugify
 from tqdm import tqdm
 
+import hearpreprocess.util.audio as audio_util
+from hearpreprocess.util.luigi import WorkTask, download_file, new_basedir
+
```
Comment (jorshi) [inline hearpreprocess/pipeline.py:20]: Nice, can disregard comment in previous PR

**Review by jorshi — APPROVED** [review]

File: requirements.txt [hunk 3]
```diff
@@ -1,29 +1,14 @@
 black
 boto3
```
Comment (jorshi) [inline requirements.txt:2]: Can we ditch boto3?

File: README.md (hunk 1 above)
Comment (turian) [reply README.md:8]: Yes, on google cloud or huggingaface

File: requirements.txt (hunk 3 above)
Comment (turian) [reply requirements.txt:2]: I think i removed it in a later PR?

**Pull request merged**